cmd/server/BUILD

```diff
@@ -9,7 +9,6 @@ go_library(
         "main.go",
         "server.go",
     ],
-    visibility = ["//visibility:private"],
```

Comment:
same question as above. we don't want things depending on this package, correct?